### PR TITLE
evan-creates pull request template 

### PR DESCRIPTION
**Pull request by easpaas**
## Description
---
This markdown file shows an outline for how the pull request will flow. 

## Link to trello card
https://trello.com/c/qn8UMI2U

### CODE REVIEW

File: PULL-REQUEST.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Pull Request Template
+
```
Comment (KyleTy1er) [inline PULL-REQUEST.md:2]: This is a sentence made up of letters.

**Review by KyleTy1er — APPROVED** [review]

File: PULL-REQUEST.md (hunk 1 above)
Comment (easpaas) [reply PULL-REQUEST.md:2]: Correct! Forgot to add the content, I will add that now.

**Review by dougscohen — APPROVED** [review]
be sure to add commit naming convention and other stuff that got deleted. other than that looks good!

might be good to eventually add a screenshot of what an example would look like.

**Comment by easpaas** [comment]
creating a pull request

**Review by KyleTy1er — APPROVED** [review]

File: PULL-REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Pull Request Template
+
+**Person who creates PR SHOULD NOT merge own pull request**
```
Comment (dougscohen) [inline PULL-REQUEST.md:3]: I would say don't use PR and pull request. Use one or the other. So in this case maybe it should say:

The person who creates the pull request SHOULD NOT merge their own pull request**

File: PULL-REQUEST.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Commits should follow the format below,
+```
+git commit -m 'FIX: readme'
+git commit -m 'FEAT: initial navbar component'
+git commit -m 'MISC: re-organizing repo for docker build'
+```
+
+## Reviewers
+- Each pull request should include a minimum of 2 reviewers
+- Reviewers should comment a minimum of 2 constructive sentences
```
Comment (dougscohen) [inline PULL-REQUEST.md:25]: Maybe put: Last reviewer should merge if everyone has approved changes.

**Review by dougscohen — COMMENTED** [review]
Made a couple of comments. Take a look and then once fixed it I'll merge!

File: PULL-REQUEST.md (hunk 3 above)
Comment (easpaas) [reply PULL-REQUEST.md:25]: Just saw this...adding this to file now 👍

File: PULL-REQUEST.md (hunk 2 above)
Comment (easpaas) [reply PULL-REQUEST.md:3]: ✔️  Great catch. Changing this now

**Review by dougscohen — APPROVED** [review]
Thanks for your hard work, Evan! no more complaints from me!

**Review by dougscohen — COMMENTED** [review]
@KyleTy1er please review again and merge if it looks good!
